[2022-06-23 18:37:23,892] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: match_details_dag.dbt_test scheduled__2022-06-20T00:00:00+00:00 [queued]>
[2022-06-23 18:37:23,901] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: match_details_dag.dbt_test scheduled__2022-06-20T00:00:00+00:00 [queued]>
[2022-06-23 18:37:23,901] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-06-23 18:37:23,902] {taskinstance.py:1244} INFO - Starting attempt 1 of 2
[2022-06-23 18:37:23,902] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-06-23 18:37:23,911] {taskinstance.py:1264} INFO - Executing <Task(BashOperator): dbt_test> on 2022-06-20 00:00:00+00:00
[2022-06-23 18:37:23,916] {standard_task_runner.py:52} INFO - Started process 369 to run task
[2022-06-23 18:37:23,919] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'match_details_dag', 'dbt_test', 'scheduled__2022-06-20T00:00:00+00:00', '--job-id', '285', '--raw', '--subdir', 'DAGS_FOLDER/matchData_dag.py', '--cfg-path', '/tmp/tmpmp42bl47', '--error-file', '/tmp/tmpga6gdh0b']
[2022-06-23 18:37:23,919] {standard_task_runner.py:77} INFO - Job 285: Subtask dbt_test
[2022-06-23 18:37:23,961] {logging_mixin.py:109} INFO - Running <TaskInstance: match_details_dag.dbt_test scheduled__2022-06-20T00:00:00+00:00 [running]> on host ebe3cc7dbee2
[2022-06-23 18:37:24,004] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=match_details_dag
AIRFLOW_CTX_TASK_ID=dbt_test
AIRFLOW_CTX_EXECUTION_DATE=2022-06-20T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-20T00:00:00+00:00
[2022-06-23 18:37:24,004] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-23 18:37:24,005] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'cd /opt/***/dbt && dbt test']
[2022-06-23 18:37:24,015] {subprocess.py:85} INFO - Output:
[2022-06-23 18:37:27,473] {subprocess.py:89} INFO - 18:37:27  Running with dbt=1.1.0
[2022-06-23 18:37:27,557] {subprocess.py:89} INFO - 18:37:27  Found 17 models, 32 tests, 0 snapshots, 0 analyses, 400 macros, 0 operations, 0 seed files, 24 sources, 0 exposures, 0 metrics
[2022-06-23 18:37:27,562] {subprocess.py:89} INFO - 18:37:27
[2022-06-23 18:37:28,422] {subprocess.py:89} INFO - 18:37:28  Concurrency: 10 threads (target='dev')
[2022-06-23 18:37:28,423] {subprocess.py:89} INFO - 18:37:28
[2022-06-23 18:37:28,444] {subprocess.py:89} INFO - 18:37:28  1 of 32 START test not_null_final_champions_champion_id ........................ [RUN]
[2022-06-23 18:37:28,445] {subprocess.py:89} INFO - 18:37:28  2 of 32 START test not_null_final_items_item_id ................................ [RUN]
[2022-06-23 18:37:28,447] {subprocess.py:89} INFO - 18:37:28  3 of 32 START test not_null_final_maps_map_id .................................. [RUN]
[2022-06-23 18:37:28,449] {subprocess.py:89} INFO - 18:37:28  4 of 32 START test not_null_final_match_details_matchId ........................ [RUN]
[2022-06-23 18:37:28,450] {subprocess.py:89} INFO - 18:37:28  5 of 32 START test not_null_final_match_info_matchId ........................... [RUN]
[2022-06-23 18:37:28,450] {subprocess.py:89} INFO - 18:37:28  6 of 32 START test not_null_final_players_summonerId ........................... [RUN]
[2022-06-23 18:37:28,451] {subprocess.py:89} INFO - 18:37:28  7 of 32 START test not_null_final_runes_reforged_rune_id ....................... [RUN]
[2022-06-23 18:37:28,452] {subprocess.py:89} INFO - 18:37:28  8 of 32 START test not_null_final_summonerspells_summonerspell_id .............. [RUN]
[2022-06-23 18:37:28,453] {subprocess.py:89} INFO - 18:37:28  9 of 32 START test not_null_stg_champions_champion_id .......................... [RUN]
[2022-06-23 18:37:28,454] {subprocess.py:89} INFO - 18:37:28  10 of 32 START test not_null_stg_items_item_id ................................. [RUN]
[2022-06-23 18:37:30,445] {subprocess.py:89} INFO - 18:37:30  1 of 32 PASS not_null_final_champions_champion_id .............................. [[32mPASS[0m in 2.00s]
[2022-06-23 18:37:30,451] {subprocess.py:89} INFO - 18:37:30  11 of 32 START test not_null_stg_maps_map_id ................................... [RUN]
[2022-06-23 18:37:32,282] {subprocess.py:89} INFO - 18:37:32  2 of 32 PASS not_null_final_items_item_id ...................................... [[32mPASS[0m in 3.84s]
[2022-06-23 18:37:32,288] {subprocess.py:89} INFO - 18:37:32  12 of 32 START test not_null_stg_match_details_matchId ......................... [RUN]
[2022-06-23 18:37:32,671] {subprocess.py:89} INFO - 18:37:32  9 of 32 PASS not_null_stg_champions_champion_id ................................ [[32mPASS[0m in 4.20s]
[2022-06-23 18:37:32,712] {subprocess.py:89} INFO - 18:37:32  13 of 32 START test not_null_stg_match_info_matchId ............................ [RUN]
[2022-06-23 18:37:32,787] {subprocess.py:89} INFO - 18:37:32  3 of 32 PASS not_null_final_maps_map_id ........................................ [[32mPASS[0m in 4.34s]
[2022-06-23 18:37:32,791] {subprocess.py:89} INFO - 18:37:32  4 of 32 PASS not_null_final_match_details_matchId .............................. [[32mPASS[0m in 4.34s]
[2022-06-23 18:37:32,801] {subprocess.py:89} INFO - 18:37:32  10 of 32 PASS not_null_stg_items_item_id ....................................... [[32mPASS[0m in 4.34s]
[2022-06-23 18:37:32,804] {subprocess.py:89} INFO - 18:37:32  14 of 32 START test not_null_stg_players_summonerId ............................ [RUN]
[2022-06-23 18:37:32,807] {subprocess.py:89} INFO - 18:37:32  6 of 32 PASS not_null_final_players_summonerId ................................. [[32mPASS[0m in 4.35s]
[2022-06-23 18:37:32,811] {subprocess.py:89} INFO - 18:37:32  15 of 32 START test not_null_stg_runes_reforged_rune_id ........................ [RUN]
[2022-06-23 18:37:32,812] {subprocess.py:89} INFO - 18:37:32  8 of 32 PASS not_null_final_summonerspells_summonerspell_id .................... [[32mPASS[0m in 4.35s]
[2022-06-23 18:37:32,815] {subprocess.py:89} INFO - 18:37:32  5 of 32 PASS not_null_final_match_info_matchId ................................. [[32mPASS[0m in 4.36s]
[2022-06-23 18:37:32,823] {subprocess.py:89} INFO - 18:37:32  11 of 32 PASS not_null_stg_maps_map_id ......................................... [[32mPASS[0m in 2.36s]
[2022-06-23 18:37:32,827] {subprocess.py:89} INFO - 18:37:32  16 of 32 START test not_null_stg_summonerspells_summonerspell_id ............... [RUN]
[2022-06-23 18:37:32,829] {subprocess.py:89} INFO - 18:37:32  7 of 32 PASS not_null_final_runes_reforged_rune_id ............................. [[32mPASS[0m in 4.37s]
[2022-06-23 18:37:32,839] {subprocess.py:89} INFO - 18:37:32  17 of 32 START test unique_final_champions_champion_id ......................... [RUN]
[2022-06-23 18:37:32,847] {subprocess.py:89} INFO - 18:37:32  18 of 32 START test unique_final_items_item_id ................................. [RUN]
[2022-06-23 18:37:32,851] {subprocess.py:89} INFO - 18:37:32  19 of 32 START test unique_final_maps_map_id ................................... [RUN]
[2022-06-23 18:37:32,856] {subprocess.py:89} INFO - 18:37:32  20 of 32 START test unique_final_match_details_matchId ......................... [RUN]
[2022-06-23 18:37:32,866] {subprocess.py:89} INFO - 18:37:32  21 of 32 START test unique_final_match_info_matchId ............................ [RUN]
[2022-06-23 18:37:33,450] {subprocess.py:89} INFO - 18:37:33  12 of 32 PASS not_null_stg_match_details_matchId ............................... [[32mPASS[0m in 1.14s]
[2022-06-23 18:37:33,524] {subprocess.py:89} INFO - 18:37:33  22 of 32 START test unique_final_players_summonerId ............................ [RUN]
[2022-06-23 18:37:33,879] {subprocess.py:89} INFO - 18:37:33  13 of 32 PASS not_null_stg_match_info_matchId .................................. [[32mPASS[0m in 1.16s]
[2022-06-23 18:37:33,885] {subprocess.py:89} INFO - 18:37:33  23 of 32 START test unique_final_runes_reforged_rune_id ........................ [RUN]
[2022-06-23 18:37:34,798] {subprocess.py:89} INFO - 18:37:34  14 of 32 PASS not_null_stg_players_summonerId .................................. [[32mPASS[0m in 1.99s]
[2022-06-23 18:37:34,808] {subprocess.py:89} INFO - 18:37:34  24 of 32 START test unique_final_summonerspells_summonerspell_id ............... [RUN]
[2022-06-23 18:37:35,692] {subprocess.py:89} INFO - 18:37:35  16 of 32 PASS not_null_stg_summonerspells_summonerspell_id ..................... [[32mPASS[0m in 2.86s]
[2022-06-23 18:37:35,700] {subprocess.py:89} INFO - 18:37:35  25 of 32 START test unique_stg_champions_champion_id ........................... [RUN]
[2022-06-23 18:37:35,850] {subprocess.py:89} INFO - 18:37:35  15 of 32 PASS not_null_stg_runes_reforged_rune_id .............................. [[32mPASS[0m in 3.03s]
[2022-06-23 18:37:35,871] {subprocess.py:89} INFO - 18:37:35  26 of 32 START test unique_stg_items_item_id ................................... [RUN]
[2022-06-23 18:37:37,713] {subprocess.py:89} INFO - 18:37:37  20 of 32 FAIL 111 unique_final_match_details_matchId ........................... [[31mFAIL 111[0m in 4.86s]
[2022-06-23 18:37:37,716] {subprocess.py:89} INFO - 18:37:37  27 of 32 START test unique_stg_maps_map_id ..................................... [RUN]
[2022-06-23 18:37:37,809] {subprocess.py:89} INFO - 18:37:37  21 of 32 PASS unique_final_match_info_matchId .................................. [[32mPASS[0m in 4.94s]
[2022-06-23 18:37:37,811] {subprocess.py:89} INFO - 18:37:37  28 of 32 START test unique_stg_match_details_matchId ........................... [RUN]
[2022-06-23 18:37:37,840] {subprocess.py:89} INFO - 18:37:37  18 of 32 PASS unique_final_items_item_id ....................................... [[32mPASS[0m in 4.99s]
[2022-06-23 18:37:37,841] {subprocess.py:89} INFO - 18:37:37  22 of 32 PASS unique_final_players_summonerId .................................. [[32mPASS[0m in 4.31s]
[2022-06-23 18:37:37,845] {subprocess.py:89} INFO - 18:37:37  29 of 32 START test unique_stg_match_info_matchId .............................. [RUN]
[2022-06-23 18:37:37,846] {subprocess.py:89} INFO - 18:37:37  30 of 32 START test unique_stg_players_summonerId .............................. [RUN]
[2022-06-23 18:37:37,881] {subprocess.py:89} INFO - 18:37:37  19 of 32 PASS unique_final_maps_map_id ......................................... [[32mPASS[0m in 5.03s]
[2022-06-23 18:37:37,882] {subprocess.py:89} INFO - 18:37:37  17 of 32 PASS unique_final_champions_champion_id ............................... [[32mPASS[0m in 5.04s]
[2022-06-23 18:37:37,900] {subprocess.py:89} INFO - 18:37:37  31 of 32 START test unique_stg_runes_reforged_rune_id .......................... [RUN]
[2022-06-23 18:37:37,902] {subprocess.py:89} INFO - 18:37:37  32 of 32 START test unique_stg_summonerspells_summonerspell_id ................. [RUN]
[2022-06-23 18:37:38,242] {subprocess.py:89} INFO - 18:37:38  23 of 32 FAIL 5 unique_final_runes_reforged_rune_id ............................ [[31mFAIL 5[0m in 4.25s]
[2022-06-23 18:37:38,325] {subprocess.py:89} INFO - 18:37:38  24 of 32 PASS unique_final_summonerspells_summonerspell_id ..................... [[32mPASS[0m in 3.49s]
[2022-06-23 18:37:38,379] {subprocess.py:89} INFO - 18:37:38  25 of 32 PASS unique_stg_champions_champion_id ................................. [[32mPASS[0m in 2.63s]
[2022-06-23 18:37:38,617] {subprocess.py:89} INFO - 18:37:38  26 of 32 PASS unique_stg_items_item_id ......................................... [[32mPASS[0m in 2.71s]
[2022-06-23 18:37:40,204] {subprocess.py:89} INFO - 18:37:40  28 of 32 FAIL 100 unique_stg_match_details_matchId ............................. [[31mFAIL 100[0m in 2.39s]
[2022-06-23 18:37:40,289] {subprocess.py:89} INFO - 18:37:40  30 of 32 PASS unique_stg_players_summonerId .................................... [[32mPASS[0m in 2.44s]
[2022-06-23 18:37:40,308] {subprocess.py:89} INFO - 18:37:40  31 of 32 FAIL 5 unique_stg_runes_reforged_rune_id .............................. [[31mFAIL 5[0m in 2.41s]
[2022-06-23 18:37:40,312] {subprocess.py:89} INFO - 18:37:40  27 of 32 PASS unique_stg_maps_map_id ........................................... [[32mPASS[0m in 2.59s]
[2022-06-23 18:37:40,420] {subprocess.py:89} INFO - 18:37:40  32 of 32 PASS unique_stg_summonerspells_summonerspell_id ....................... [[32mPASS[0m in 2.52s]
[2022-06-23 18:37:40,442] {subprocess.py:89} INFO - 18:37:40  29 of 32 PASS unique_stg_match_info_matchId .................................... [[32mPASS[0m in 2.60s]
[2022-06-23 18:37:40,447] {subprocess.py:89} INFO - 18:37:40
[2022-06-23 18:37:40,448] {subprocess.py:89} INFO - 18:37:40  Finished running 32 tests in 12.88s.
[2022-06-23 18:37:40,470] {subprocess.py:89} INFO - 18:37:40
[2022-06-23 18:37:40,470] {subprocess.py:89} INFO - 18:37:40  [31mCompleted with 4 errors and 0 warnings:[0m
[2022-06-23 18:37:40,471] {subprocess.py:89} INFO - 18:37:40
[2022-06-23 18:37:40,471] {subprocess.py:89} INFO - 18:37:40  [31mFailure in test unique_final_match_details_matchId (models/warehouse/schema.yml)[0m
[2022-06-23 18:37:40,472] {subprocess.py:89} INFO - 18:37:40    Got 111 results, configured to fail if != 0
[2022-06-23 18:37:40,472] {subprocess.py:89} INFO - 18:37:40
[2022-06-23 18:37:40,472] {subprocess.py:89} INFO - 18:37:40    compiled SQL at target/compiled/leagueoflegendsdbt/models/warehouse/schema.yml/unique_final_match_details_matchId.sql
[2022-06-23 18:37:40,473] {subprocess.py:89} INFO - 18:37:40
[2022-06-23 18:37:40,474] {subprocess.py:89} INFO - 18:37:40  [31mFailure in test unique_final_runes_reforged_rune_id (models/warehouse/schema.yml)[0m
[2022-06-23 18:37:40,474] {subprocess.py:89} INFO - 18:37:40    Got 5 results, configured to fail if != 0
[2022-06-23 18:37:40,475] {subprocess.py:89} INFO - 18:37:40
[2022-06-23 18:37:40,475] {subprocess.py:89} INFO - 18:37:40    compiled SQL at target/compiled/leagueoflegendsdbt/models/warehouse/schema.yml/unique_final_runes_reforged_rune_id.sql
[2022-06-23 18:37:40,475] {subprocess.py:89} INFO - 18:37:40
[2022-06-23 18:37:40,476] {subprocess.py:89} INFO - 18:37:40  [31mFailure in test unique_stg_match_details_matchId (models/staging/schema.yml)[0m
[2022-06-23 18:37:40,476] {subprocess.py:89} INFO - 18:37:40    Got 100 results, configured to fail if != 0
[2022-06-23 18:37:40,477] {subprocess.py:89} INFO - 18:37:40
[2022-06-23 18:37:40,477] {subprocess.py:89} INFO - 18:37:40    compiled SQL at target/compiled/leagueoflegendsdbt/models/staging/schema.yml/unique_stg_match_details_matchId.sql
[2022-06-23 18:37:40,477] {subprocess.py:89} INFO - 18:37:40
[2022-06-23 18:37:40,478] {subprocess.py:89} INFO - 18:37:40  [31mFailure in test unique_stg_runes_reforged_rune_id (models/staging_datadragon/schema.yml)[0m
[2022-06-23 18:37:40,478] {subprocess.py:89} INFO - 18:37:40    Got 5 results, configured to fail if != 0
[2022-06-23 18:37:40,479] {subprocess.py:89} INFO - 18:37:40
[2022-06-23 18:37:40,479] {subprocess.py:89} INFO - 18:37:40    compiled SQL at target/compiled/leagueoflegendsdbt/models/staging_datadragon/schema.yml/unique_stg_runes_reforged_rune_id.sql
[2022-06-23 18:37:40,480] {subprocess.py:89} INFO - 18:37:40
[2022-06-23 18:37:40,480] {subprocess.py:89} INFO - 18:37:40  Done. PASS=28 WARN=0 ERROR=4 SKIP=0 TOTAL=32
[2022-06-23 18:37:41,174] {subprocess.py:93} INFO - Command exited with return code 1
[2022-06-23 18:37:41,188] {taskinstance.py:1718} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 187, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2022-06-23 18:37:41,193] {taskinstance.py:1272} INFO - Marking task as UP_FOR_RETRY. dag_id=match_details_dag, task_id=dbt_test, execution_date=20220620T000000, start_date=20220623T183723, end_date=20220623T183741
[2022-06-23 18:37:41,205] {standard_task_runner.py:89} ERROR - Failed to execute job 285 for task dbt_test
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 180, in _run_raw_task
    ti._run_raw_task(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 187, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2022-06-23 18:37:41,241] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-06-23 18:37:41,270] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-23 21:21:16,436] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: match_details_dag.dbt_test scheduled__2022-06-20T00:00:00+00:00 [queued]>
[2022-06-23 21:21:16,447] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: match_details_dag.dbt_test scheduled__2022-06-20T00:00:00+00:00 [queued]>
[2022-06-23 21:21:16,447] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-06-23 21:21:16,447] {taskinstance.py:1244} INFO - Starting attempt 1 of 2
[2022-06-23 21:21:16,447] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-06-23 21:21:16,458] {taskinstance.py:1264} INFO - Executing <Task(BashOperator): dbt_test> on 2022-06-20 00:00:00+00:00
[2022-06-23 21:21:16,464] {standard_task_runner.py:52} INFO - Started process 1240 to run task
[2022-06-23 21:21:16,467] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'match_details_dag', 'dbt_test', 'scheduled__2022-06-20T00:00:00+00:00', '--job-id', '298', '--raw', '--subdir', 'DAGS_FOLDER/matchData_dag.py', '--cfg-path', '/tmp/tmpm7uxeifb', '--error-file', '/tmp/tmp709asqfk']
[2022-06-23 21:21:16,468] {standard_task_runner.py:77} INFO - Job 298: Subtask dbt_test
[2022-06-23 21:21:16,523] {logging_mixin.py:109} INFO - Running <TaskInstance: match_details_dag.dbt_test scheduled__2022-06-20T00:00:00+00:00 [running]> on host ebe3cc7dbee2
[2022-06-23 21:21:16,569] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=match_details_dag
AIRFLOW_CTX_TASK_ID=dbt_test
AIRFLOW_CTX_EXECUTION_DATE=2022-06-20T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-20T00:00:00+00:00
[2022-06-23 21:21:16,570] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-23 21:21:16,571] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'cd /opt/***/dbt && dbt test']
[2022-06-23 21:21:16,583] {subprocess.py:85} INFO - Output:
[2022-06-23 21:21:20,762] {subprocess.py:89} INFO - 21:21:20  Running with dbt=1.1.0
[2022-06-23 21:21:20,876] {subprocess.py:89} INFO - 21:21:20  Found 17 models, 32 tests, 0 snapshots, 0 analyses, 400 macros, 0 operations, 0 seed files, 24 sources, 0 exposures, 0 metrics
[2022-06-23 21:21:20,885] {subprocess.py:89} INFO - 21:21:20
[2022-06-23 21:21:21,835] {subprocess.py:89} INFO - 21:21:21  Concurrency: 10 threads (target='dev')
[2022-06-23 21:21:21,835] {subprocess.py:89} INFO - 21:21:21
[2022-06-23 21:21:21,859] {subprocess.py:89} INFO - 21:21:21  1 of 32 START test not_null_final_champions_champion_id ........................ [RUN]
[2022-06-23 21:21:21,862] {subprocess.py:89} INFO - 21:21:21  2 of 32 START test not_null_final_items_item_id ................................ [RUN]
[2022-06-23 21:21:21,866] {subprocess.py:89} INFO - 21:21:21  3 of 32 START test not_null_final_maps_map_id .................................. [RUN]
[2022-06-23 21:21:21,867] {subprocess.py:89} INFO - 21:21:21  4 of 32 START test not_null_final_match_details_matchId ........................ [RUN]
[2022-06-23 21:21:21,868] {subprocess.py:89} INFO - 21:21:21  5 of 32 START test not_null_final_match_info_matchId ........................... [RUN]
[2022-06-23 21:21:21,870] {subprocess.py:89} INFO - 21:21:21  6 of 32 START test not_null_final_players_summonerId ........................... [RUN]
[2022-06-23 21:21:21,871] {subprocess.py:89} INFO - 21:21:21  7 of 32 START test not_null_final_runes_reforged_rune_id ....................... [RUN]
[2022-06-23 21:21:21,872] {subprocess.py:89} INFO - 21:21:21  8 of 32 START test not_null_final_summonerspells_summonerspell_id .............. [RUN]
[2022-06-23 21:21:21,873] {subprocess.py:89} INFO - 21:21:21  9 of 32 START test not_null_stg_champions_champion_id .......................... [RUN]
[2022-06-23 21:21:21,874] {subprocess.py:89} INFO - 21:21:21  10 of 32 START test not_null_stg_items_item_id ................................. [RUN]
[2022-06-23 21:21:24,670] {subprocess.py:89} INFO - 21:21:24  1 of 32 PASS not_null_final_champions_champion_id .............................. [[32mPASS[0m in 2.80s]
[2022-06-23 21:21:24,702] {subprocess.py:89} INFO - 21:21:24  11 of 32 START test not_null_stg_maps_map_id ................................... [RUN]
[2022-06-23 21:21:26,616] {subprocess.py:89} INFO - 21:21:26  5 of 32 PASS not_null_final_match_info_matchId ................................. [[32mPASS[0m in 4.75s]
[2022-06-23 21:21:26,618] {subprocess.py:89} INFO - 21:21:26  12 of 32 START test not_null_stg_match_details_matchId ......................... [RUN]
[2022-06-23 21:21:26,693] {subprocess.py:89} INFO - 21:21:26  7 of 32 PASS not_null_final_runes_reforged_rune_id ............................. [[32mPASS[0m in 4.82s]
[2022-06-23 21:21:26,696] {subprocess.py:89} INFO - 21:21:26  13 of 32 START test not_null_stg_match_info_matchId ............................ [RUN]
[2022-06-23 21:21:26,817] {subprocess.py:89} INFO - 21:21:26  6 of 32 PASS not_null_final_players_summonerId ................................. [[32mPASS[0m in 4.94s]
[2022-06-23 21:21:26,848] {subprocess.py:89} INFO - 21:21:26  14 of 32 START test not_null_stg_players_summonerId ............................ [RUN]
[2022-06-23 21:21:27,817] {subprocess.py:89} INFO - 21:21:27  2 of 32 PASS not_null_final_items_item_id ...................................... [[32mPASS[0m in 5.95s]
[2022-06-23 21:21:27,819] {subprocess.py:89} INFO - 21:21:27  15 of 32 START test not_null_stg_runes_reforged_rune_id ........................ [RUN]
[2022-06-23 21:21:27,825] {subprocess.py:89} INFO - 21:21:27  11 of 32 PASS not_null_stg_maps_map_id ......................................... [[32mPASS[0m in 3.12s]
[2022-06-23 21:21:27,827] {subprocess.py:89} INFO - 21:21:27  16 of 32 START test not_null_stg_summonerspells_summonerspell_id ............... [RUN]
[2022-06-23 21:21:28,336] {subprocess.py:89} INFO - 21:21:28  8 of 32 PASS not_null_final_summonerspells_summonerspell_id .................... [[32mPASS[0m in 6.31s]
[2022-06-23 21:21:28,433] {subprocess.py:89} INFO - 21:21:28  4 of 32 PASS not_null_final_match_details_matchId .............................. [[32mPASS[0m in 6.46s]
[2022-06-23 21:21:28,498] {subprocess.py:89} INFO - 21:21:28  17 of 32 START test unique_final_champions_champion_id ......................... [RUN]
[2022-06-23 21:21:28,505] {subprocess.py:89} INFO - 21:21:28  18 of 32 START test unique_final_items_item_id ................................. [RUN]
[2022-06-23 21:21:28,543] {subprocess.py:89} INFO - 21:21:28  3 of 32 PASS not_null_final_maps_map_id ........................................ [[32mPASS[0m in 6.67s]
[2022-06-23 21:21:28,569] {subprocess.py:89} INFO - 21:21:28  19 of 32 START test unique_final_maps_map_id ................................... [RUN]
[2022-06-23 21:21:28,571] {subprocess.py:89} INFO - 21:21:28  12 of 32 PASS not_null_stg_match_details_matchId ............................... [[32mPASS[0m in 1.94s]
[2022-06-23 21:21:28,575] {subprocess.py:89} INFO - 21:21:28  9 of 32 PASS not_null_stg_champions_champion_id ................................ [[32mPASS[0m in 6.70s]
[2022-06-23 21:21:28,600] {subprocess.py:89} INFO - 21:21:28  13 of 32 PASS not_null_stg_match_info_matchId .................................. [[32mPASS[0m in 1.89s]
[2022-06-23 21:21:28,614] {subprocess.py:89} INFO - 21:21:28  20 of 32 START test unique_final_match_details_matchId ......................... [RUN]
[2022-06-23 21:21:28,619] {subprocess.py:89} INFO - 21:21:28  21 of 32 START test unique_final_match_info_matchId ............................ [RUN]
[2022-06-23 21:21:28,622] {subprocess.py:89} INFO - 21:21:28  10 of 32 PASS not_null_stg_items_item_id ....................................... [[32mPASS[0m in 6.73s]
[2022-06-23 21:21:28,624] {subprocess.py:89} INFO - 21:21:28  14 of 32 PASS not_null_stg_players_summonerId .................................. [[32mPASS[0m in 1.76s]
[2022-06-23 21:21:28,633] {subprocess.py:89} INFO - 21:21:28  22 of 32 START test unique_final_players_summonerId ............................ [RUN]
[2022-06-23 21:21:28,668] {subprocess.py:89} INFO - 21:21:28  23 of 32 START test unique_final_runes_reforged_rune_id ........................ [RUN]
[2022-06-23 21:21:28,673] {subprocess.py:89} INFO - 21:21:28  24 of 32 START test unique_final_summonerspells_summonerspell_id ............... [RUN]
[2022-06-23 21:21:30,375] {subprocess.py:89} INFO - 21:21:30  15 of 32 PASS not_null_stg_runes_reforged_rune_id .............................. [[32mPASS[0m in 2.55s]
[2022-06-23 21:21:30,395] {subprocess.py:89} INFO - 21:21:30  16 of 32 PASS not_null_stg_summonerspells_summonerspell_id ..................... [[32mPASS[0m in 2.55s]
[2022-06-23 21:21:30,431] {subprocess.py:89} INFO - 21:21:30  25 of 32 START test unique_stg_champions_champion_id ........................... [RUN]
[2022-06-23 21:21:30,469] {subprocess.py:89} INFO - 21:21:30  26 of 32 START test unique_stg_items_item_id ................................... [RUN]
[2022-06-23 21:21:30,612] {subprocess.py:89} INFO - 21:21:30  17 of 32 PASS unique_final_champions_champion_id ............................... [[32mPASS[0m in 2.11s]
[2022-06-23 21:21:30,622] {subprocess.py:89} INFO - 21:21:30  27 of 32 START test unique_stg_maps_map_id ..................................... [RUN]
[2022-06-23 21:21:30,746] {subprocess.py:89} INFO - 21:21:30  18 of 32 PASS unique_final_items_item_id ....................................... [[32mPASS[0m in 2.24s]
[2022-06-23 21:21:30,770] {subprocess.py:89} INFO - 21:21:30  28 of 32 START test unique_stg_match_details_matchId ........................... [RUN]
[2022-06-23 21:21:32,107] {subprocess.py:89} INFO - 21:21:32  19 of 32 PASS unique_final_maps_map_id ......................................... [[32mPASS[0m in 3.53s]
[2022-06-23 21:21:32,120] {subprocess.py:89} INFO - 21:21:32  29 of 32 START test unique_stg_match_info_matchId .............................. [RUN]
[2022-06-23 21:21:34,026] {subprocess.py:89} INFO - 21:21:34  20 of 32 FAIL 100 unique_final_match_details_matchId ........................... [[31mFAIL 100[0m in 5.41s]
[2022-06-23 21:21:34,031] {subprocess.py:89} INFO - 21:21:34  30 of 32 START test unique_stg_players_summonerId .............................. [RUN]
[2022-06-23 21:21:34,093] {subprocess.py:89} INFO - 21:21:34  21 of 32 PASS unique_final_match_info_matchId .................................. [[32mPASS[0m in 5.47s]
[2022-06-23 21:21:34,100] {subprocess.py:89} INFO - 21:21:34  31 of 32 START test unique_stg_runes_reforged_rune_id .......................... [RUN]
[2022-06-23 21:21:34,180] {subprocess.py:89} INFO - 21:21:34  22 of 32 PASS unique_final_players_summonerId .................................. [[32mPASS[0m in 5.54s]
[2022-06-23 21:21:34,190] {subprocess.py:89} INFO - 21:21:34  32 of 32 START test unique_stg_summonerspells_summonerspell_id ................. [RUN]
[2022-06-23 21:21:34,241] {subprocess.py:89} INFO - 21:21:34  23 of 32 FAIL 5 unique_final_runes_reforged_rune_id ............................ [[31mFAIL 5[0m in 5.57s]
[2022-06-23 21:21:34,280] {subprocess.py:89} INFO - 21:21:34  24 of 32 PASS unique_final_summonerspells_summonerspell_id ..................... [[32mPASS[0m in 5.61s]
[2022-06-23 21:21:34,799] {subprocess.py:89} INFO - 21:21:34  29 of 32 PASS unique_stg_match_info_matchId .................................... [[32mPASS[0m in 2.68s]
[2022-06-23 21:21:34,887] {subprocess.py:89} INFO - 21:21:34  26 of 32 PASS unique_stg_items_item_id ......................................... [[32mPASS[0m in 4.41s]
[2022-06-23 21:21:34,900] {subprocess.py:89} INFO - 21:21:34  25 of 32 PASS unique_stg_champions_champion_id ................................. [[32mPASS[0m in 4.45s]
[2022-06-23 21:21:34,921] {subprocess.py:89} INFO - 21:21:34  28 of 32 FAIL 100 unique_stg_match_details_matchId ............................. [[31mFAIL 100[0m in 4.14s]
[2022-06-23 21:21:34,925] {subprocess.py:89} INFO - 21:21:34  27 of 32 PASS unique_stg_maps_map_id ........................................... [[32mPASS[0m in 4.29s]
[2022-06-23 21:21:35,584] {subprocess.py:89} INFO - 21:21:35  30 of 32 PASS unique_stg_players_summonerId .................................... [[32mPASS[0m in 1.55s]
[2022-06-23 21:21:35,647] {subprocess.py:89} INFO - 21:21:35  31 of 32 FAIL 5 unique_stg_runes_reforged_rune_id .............................. [[31mFAIL 5[0m in 1.55s]
[2022-06-23 21:21:35,669] {subprocess.py:89} INFO - 21:21:35  32 of 32 PASS unique_stg_summonerspells_summonerspell_id ....................... [[32mPASS[0m in 1.47s]
[2022-06-23 21:21:35,675] {subprocess.py:89} INFO - 21:21:35
[2022-06-23 21:21:35,675] {subprocess.py:89} INFO - 21:21:35  Finished running 32 tests in 14.79s.
[2022-06-23 21:21:35,701] {subprocess.py:89} INFO - 21:21:35
[2022-06-23 21:21:35,702] {subprocess.py:89} INFO - 21:21:35  [31mCompleted with 4 errors and 0 warnings:[0m
[2022-06-23 21:21:35,702] {subprocess.py:89} INFO - 21:21:35
[2022-06-23 21:21:35,703] {subprocess.py:89} INFO - 21:21:35  [31mFailure in test unique_final_match_details_matchId (models/warehouse/schema.yml)[0m
[2022-06-23 21:21:35,703] {subprocess.py:89} INFO - 21:21:35    Got 100 results, configured to fail if != 0
[2022-06-23 21:21:35,704] {subprocess.py:89} INFO - 21:21:35
[2022-06-23 21:21:35,704] {subprocess.py:89} INFO - 21:21:35    compiled SQL at target/compiled/leagueoflegendsdbt/models/warehouse/schema.yml/unique_final_match_details_matchId.sql
[2022-06-23 21:21:35,705] {subprocess.py:89} INFO - 21:21:35
[2022-06-23 21:21:35,706] {subprocess.py:89} INFO - 21:21:35  [31mFailure in test unique_final_runes_reforged_rune_id (models/warehouse/schema.yml)[0m
[2022-06-23 21:21:35,707] {subprocess.py:89} INFO - 21:21:35    Got 5 results, configured to fail if != 0
[2022-06-23 21:21:35,708] {subprocess.py:89} INFO - 21:21:35
[2022-06-23 21:21:35,708] {subprocess.py:89} INFO - 21:21:35    compiled SQL at target/compiled/leagueoflegendsdbt/models/warehouse/schema.yml/unique_final_runes_reforged_rune_id.sql
[2022-06-23 21:21:35,709] {subprocess.py:89} INFO - 21:21:35
[2022-06-23 21:21:35,710] {subprocess.py:89} INFO - 21:21:35  [31mFailure in test unique_stg_match_details_matchId (models/staging/schema.yml)[0m
[2022-06-23 21:21:35,710] {subprocess.py:89} INFO - 21:21:35    Got 100 results, configured to fail if != 0
[2022-06-23 21:21:35,710] {subprocess.py:89} INFO - 21:21:35
[2022-06-23 21:21:35,711] {subprocess.py:89} INFO - 21:21:35    compiled SQL at target/compiled/leagueoflegendsdbt/models/staging/schema.yml/unique_stg_match_details_matchId.sql
[2022-06-23 21:21:35,711] {subprocess.py:89} INFO - 21:21:35
[2022-06-23 21:21:35,712] {subprocess.py:89} INFO - 21:21:35  [31mFailure in test unique_stg_runes_reforged_rune_id (models/staging_datadragon/schema.yml)[0m
[2022-06-23 21:21:35,713] {subprocess.py:89} INFO - 21:21:35    Got 5 results, configured to fail if != 0
[2022-06-23 21:21:35,713] {subprocess.py:89} INFO - 21:21:35
[2022-06-23 21:21:35,714] {subprocess.py:89} INFO - 21:21:35    compiled SQL at target/compiled/leagueoflegendsdbt/models/staging_datadragon/schema.yml/unique_stg_runes_reforged_rune_id.sql
[2022-06-23 21:21:35,715] {subprocess.py:89} INFO - 21:21:35
[2022-06-23 21:21:35,715] {subprocess.py:89} INFO - 21:21:35  Done. PASS=28 WARN=0 ERROR=4 SKIP=0 TOTAL=32
[2022-06-23 21:21:36,571] {subprocess.py:93} INFO - Command exited with return code 1
[2022-06-23 21:21:36,586] {taskinstance.py:1718} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 187, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2022-06-23 21:21:36,595] {taskinstance.py:1272} INFO - Marking task as UP_FOR_RETRY. dag_id=match_details_dag, task_id=dbt_test, execution_date=20220620T000000, start_date=20220623T212116, end_date=20220623T212136
[2022-06-23 21:21:36,616] {standard_task_runner.py:89} ERROR - Failed to execute job 298 for task dbt_test
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 180, in _run_raw_task
    ti._run_raw_task(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 187, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2022-06-23 21:21:36,663] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-06-23 21:21:36,703] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
